events/urls.py

```diff
@@ -4,6 +4,10 @@
 
 app_name = "events"
 
+# see views.py for a description of each URL
 urlpatterns = [
     path("", EventsIndexPage.as_view(), name="index"),
+    path("create", EventCreatePage.as_view()),
```

Comment:
Commenting here but this applies anywhere. While this is fine and perfectly acceptable (this world work, there's nothing wrong with this), I'd recommend adding a slash at the end of each url, and giving it a name. It just makes it nicer to refer to in templates as you can type `{% url 'events:create' %} to point to the url or whatever, and the slash is just something I've already done, IDK if it actually makes a difference :P Very small things though lmao
```suggestion
    path("create/", EventCreatePage.as_view(), name="create"),
```

Reply:
I’ll make all these changes in the morning, ty for pointing out